### DISCUSSION TITLE
Unconstrained waypoints

### DISCUSSION THREAD

**Discussion by rjoomen** [tesseract-robotics/tesseract_planning#395] [Q&A]
If the [@todo's the Trajopt motion planner source](https://github.com/tesseract-robotics/tesseract_planning/blob/master/tesseract_motion_planners/trajopt/src/trajopt_motion_planner.cpp#L288) are implemented, would this imply that a waypoint where the plan profile has term_type cost effectively functions as if isConstrained == false? I.e. that Trajopt can move the waypoint around during the optimization.

**Comment by Levi-Armstrong** [comment]
I would say yes but would need to look at the code to confirm
